mythical-creatures/test/ogre-test.js

```diff
@@ -0,0 +1,106 @@
+var assert = require('chai').assert;
+var { Ogre, Human } = require('../exercises/ogre');
```

Comment:
Change to `{Ogre, Human}` to match convention of existing similar mythical creatures